src/erlcloud_sns.erl

```diff
@@ -171,17 +173,17 @@ confirm_subscription(SnsEvent, Config) ->
     TopicArn = binary_to_list(proplists:get_value(<<"TopicArn">>, SnsEvent, <<>>)),
```

Comment:
Line 173 is too long:     TopicArn = binary_to_list(proplists:get_value(<<"TopicArn">>, SnsEvent, <<>>)),.